src/main/java/com/autosale/shop/model/JwtTokensDAO.java

```diff
@@ -0,0 +1,18 @@
+package com.autosale.shop.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Value;
+
+@Value
+public class JwtTokensDAO {
+    String accessToken;
+
+    String refreshToken;
+
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
```

Comment:
Why do you need to use mode here?

Reply:
Actually, we don't even need JsonCreator property if we annotated all parameters in the constructor with JsonProperty annotation